core/options.js

```diff
@@ -37,10 +38,9 @@ Blockly.Options = function(options) {
     var hasDisable = false;
     var hasSounds = false;
   } else {
-    var languageTree =
-        Blockly.Options.parseToolboxTree(options['toolbox'] || null);
-    var hasCategories = Boolean(languageTree &&
-        languageTree.getElementsByTagName('category').length);
+    var languageTree = Blockly.Options.parseToolboxTree(options['toolbox'] || null);
+    languageTree = Blockly.utils.toolbox.parseToolbox(languageTree);
```

Comment:
Hard to tell from this naming what the difference is between parseToolboxTree and toolbox.parseToolbox

Reply:
Are they both parsing, or is one parsing and the other converting? Also, why not add the parse util within the ``parseToolboxTree`` definition?

Reply:
Oh I like the word converting, will use that! An parseToolboxTree is a public method, so I was trying to not change the public API.